README.md

```diff
@@ -8,9 +9,10 @@ License: GPLv2 or later
 License URI: http://www.gnu.org/licenses/gpl-2.0.html
 
 Allows you to easily sell things using Razorpay on your WordPress website.
+```
 
-== Description ==
```

Comment:
You can't use markdown here. See [this](https://wordpress.org/plugins/files/2016/06/readme.txt) and the [validator](https://wordpress.org/plugins/developers/readme-validator/)